Automattic-Tracks-iOS/Crash Logging/CrashLogging.swift

```diff
@@ -278,6 +286,104 @@ extension CrashLogging {
     }
 }
 
+// MARK: - Helpers for hybrid SDKs
+extension CrashLogging {
+    /// Returns the options required to initialize Sentry in other platforms.
+    public func getOptionsDict() -> [String: Any] {
+        return [
+            "dsn": self.dataProvider.sentryDSN,
+            "environment": self.dataProvider.buildType,
+            "releaseName": self.dataProvider.releaseName
+        ]
+    }
+
+    /// Return the current Sentry user.
+    /// This helper allows events triggered by other platforms to include the current user.
+    public func getSentryUserDict() -> [String: Any]? {
+        return dataProvider.currentUser?.sentryUser.serialize()
+    }
```

Comment:
Similar to `attachScopeToEvent`, I exposed the Sentry user for including it in the events generated in the React native side (Gutenberg).